Sources/PostgresNIO/New/Messages/Parse.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 08.01.21.
+//
+
+extension PSQLFrontendMessage {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension PSQLFrontendMessage {
```